python/tests/test_ephemeris.py

```diff
@@ -44,6 +46,12 @@ def test_sat_state():
                                   'af1': 3.637978807091713e-12,
                                   'af2': 0.0}})
   assert eph.good(g.GpsTime(**{ 'wn': 1867, 'tow': 518400.0,}))
+  eph.healthy = 0
```

Comment:
I can't believe I'd say this but maybe we should bite the bullet and migrate to using Swig.